azurerm/internal/services/apimanagement/resource_arm_api_management.go

```diff
@@ -201,6 +202,23 @@ func resourceArmApiManagementService() *schema.Resource {
 				},
 			},
 
+			"protocols": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+
```

Comment:
I have a feeling the linting will want this line removed?

```suggestion
```

Reply:
Removed line